File: docs/vs-2015/snippets/csharp/VS_Snippets_VSSDK/vssdksupportforoptionspages/cs/optionspagecustom.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.VisualStudio.Shell;
using System.ComponentModel;

namespace Microsoft.VSSDKSupportForOptionsPages
{
    class OptionsPageCustom : DialogPage
    {
        //<Snippet3>
        [Category("My Options")]
        [DisplayName("Integer Option")]
        [Description("My integer option")]
        public int OptionInteger { get; set; }
        //</Snippet3>
    }
}

File: docs/vs-2015/snippets/visualbasic/VS_Snippets_VSSDK/vssdksupportforoptionspages/vb/optionspagecustom.vb
Imports Microsoft.VisualStudio.Shell
Imports System.ComponentModel

Class OptionsPageCustom
    Inherits DialogPage

    Private _optionInt As Integer

    <Category("My Options"),
    DisplayName("Integer Option"),
    Description("My integer option")>
    Public Property OptionInteger() As Integer
        Get
            Return _optionInt
        End Get
        Set(ByVal value As Integer)
            _optionInt = value
        End Set
    End Property
End Class
